crates/amaru-ledger/src/state/volatile_db.rs

```diff
@@ -278,7 +279,8 @@ impl AnchoredVolatileState {
                                     ProposalId::from(proposal_id),
                                     proposals::Value {
                                         proposed_in,
-                                        valid_until: epoch + GOV_ACTION_LIFETIME,
+                                        valid_until: epoch
+                                            + protocol_parameters.gov_action_lifetime as Epoch,
```

Comment:
TODO will be improved in a subsequent PR by introducing `Epoch` and `EpochInterval` struct.